e2e/tests/transfer/authz_test.go

```diff
@@ -307,10 +300,9 @@ func (suite *AuthzTransferTestSuite) TestAuthz_InvalidTransferAuthorizations() {
 				Msgs:    []*codectypes.Any{protoAny},
 			}
 
-			resp, err := suite.BroadcastMessages(context.TODO(), chainA, granteeWallet, msgExec)
+			resp := suite.BroadcastMessages(context.TODO(), chainA, granteeWallet, msgExec)
 			suite.Require().NotEqual(0, resp.Code)
 			suite.Require().Contains(resp.RawLog, sdkerrors.ErrInvalidAddress.Error())
```

Comment:
Kind of. In the last SDK release, they moved their errors to be internal and recommended that each module define it's own errors rather than using SDK base errors, so we copied over only the SDK errors that were applicable to us. Since they are technically "new" errors, we gave them different error codes to have an ordering that was logical (rather than having an error list with gaps in between the codes)

I'm not sure how I feel about having error codes that cannot be referenced externally. I'm not sure I see the benefit of our errors being internal? But, I think we should leave as is and worry about it another day if another issue arises?

Reply:
> In the last SDK release, they moved their errors to be internal and recommended that each module define it's own errors rather than using SDK base errors

can't seem to be able to find this in the sdk :disappointed: . I see most of the module errors being defined, typically, in their `types` directory and sdk errors in `types/errors`. If you have time to give more context I'd love that.

> I'm not sure how I feel about having error codes that cannot be referenced externally. I'm not sure I see the benefit of our errors being internal?  But, I think we should leave as is and worry about it another day if another issue arises?

I'd assume these should be part of the API since they are returned by public facing functions, happy to leave as-is if you think that's fine atm.

Reply:
See #2896:
> We did not move the default error types over cause the idea is that users/app devs will create their own with their custom codespace.

Reply:
> I'd assume these should be part of the API since they are returned by public facing functions, happy to leave as-is if you think that's fine atm.

I think I agree, but maybe we could address in a separate issue. Would love to get @damiannolan's and @chatton's opinions here. We could open an issue and discuss at a future point? Just to have an explicit decision?

Reply:
Yup, will open an issue on this.